src/Development/Shake/Command.hs

```diff
@@ -154,6 +134,8 @@ commandExplicit funcName copts results exe args = do
                             , any (`isPrefixOf` x) inside
                             , not $ any ($ x) ignore]
                 rel = map (makeRelative cwd)
+            when (Autodeps `elem` copts) $
+              needed $ rel $ ham rs
```

Comment:
4 space indents are preferred, although not really a big deal (but I would reindent it if I'm ever near that line).